src/selectors.js

```diff
@@ -28,7 +28,7 @@ const getPolyglotOwnPhrases = (state, { ownPhrases = '' }) => (
 
 const getPolyglotOptions = (state, { polyglotOptions }) => polyglotOptions;
 
-const getPolyglot = createSelector(
+const getPolyglot = () => createSelector(
```

Comment:
Factories for selectors usually start with `create`, in this case it should be `createGetPolyglot`.
While these selectors are internal, I would rename them.